docs/modules/publisher.md

```diff
@@ -109,8 +109,8 @@ To persist the security configuration of OPC Publisher across restarts, the cert
 
 ### Configuration via Configuration File
 
-The simplest way to configure OPC Publisher is via a configuration file. An example configuration file is provided via the file [`publishednodes.json`](publishednodes.json).
-Configuration file syntax has changed over time and OPC Publisher still can read old formats, but converts them into the latest format when persisting the configuration, done regularly in an automated fashion.
+The simplest way to configure OPC Publisher is via a configuration file. An example configuration file is provided via the file [`publishednodes.json`](https://raw.githubusercontent.com/Azure/iot-edge-opc-publisher/main/opcpublisher/publishednodes.json).
```

Comment:
Let's keep the previous link to just `publishednodes.json` as `iot-edge-opc-publisher` repo is archived and it redirects to `Industrial-IoT` repo.

Reply:
Do you still want us to update link to just `publishednodes.json`?